src/listeners.js

```diff
@@ -25,4 +25,9 @@ export function registerListeners() {
 
     geb('gameBoardSelector').addEventListener('change', e_boardSelectChange);
     geb('undoButton').addEventListener('click', undoLastScore);
+
+    // Strict Mode: set winBalls to 11 and disable the field when checked
+    const strictModeCheckbox = geb('strictMode');
+    const winBallsField = geb('winBalls');
+    strictModeCheckbox.addEventListener('change', () => e_strictModeChange(strictModeCheckbox, winBallsField));
```

Comment:
这里我觉得提前查找元素意义不大，元素完全可以在事件处理函数里查找，`listeners.js`我本意就是当一个工具人用来注册函数的

就类似于这样（`e_strictModeChange`的逻辑还得改）
```suggestion
    geb('strictMode').addEventListener('change', e_strictModeChange);
```

Reply:
话说你这个命名方式我之前没见过 geb..

Reply:
> 话说你这个命名方式我之前没见过 geb..

首字母🤣

我也觉得这个命名不怎么样

但是它短啊（